Getting started guide (#481)
ScottLogic/datahelix: README.md

You can download the generator from the [GitHub project releases page](https://github.com/ScottLogic/data-engineering-generator/releases/).

> For an in-depth guide on how the generator may be used see the [step by step instructions](./docs/GettingStarted/StepByStepInstructions.md).

Reviewer: Suggest that maybe the 'getting started guide' shouldn't be 'in depth'?

```suggestion
For a guide on how the generator may be used see the [step by step instructions](./docs/GettingStarted/StepByStepInstructions.md).
```
